include/deal.II/lac/la_parallel_vector.h

```diff
@@ -354,18 +365,29 @@ namespace LinearAlgebra
 
       /**
        * Initialize the vector given to the parallel partitioning described in
-       * @p partitioner. The input argument is a shared pointer, which store
+       * @p partitioner. The input argument is a shared pointer, which stores
        * the partitioner data only once and share it between several vectors
        * with the same layout.
+       *
+       * The optional argument @p comm_sm, which consists of processes on
+       * the same shared-memory domain, allows users have read-only access to
+       * both locally-owned and ghost values of processes combined in the
+       * shared-memory communicator.
```

Comment:
Can you document how I get a MPI_Comm like this? Is there an easy way to do that?

Reply:
@peterrum do you want to extend the documentation here or in a follow-up PR with this info?